src/test/java/com/sap/oss/phosphor/fosstars/model/score/oss/ArtifactReleaseHistoryScoreTest.java

```diff
@@ -52,8 +57,8 @@ public void testCalculateVersionStatistics() {
         new ArtifactVersion("1.0.0", now.minusDays(250))));
 
     VersionStats stats = ArtifactReleaseHistoryScore.calculateVersionStats(versionInfos);
-    Assert.assertEquals(173.33333333333334, stats.averageDaysBetweenReleases, DELTA);
-    Assert.assertEquals(0.6666666666666666, stats.releaseCycleTrend, DELTA);
+    assertEquals(173.33333333333334, stats.averageDaysBetweenReleases, DELTA);
```

Comment:
`DELTA` is only 0.2.
```suggestion
    assertEquals(173.33, stats.averageDaysBetweenReleases, DELTA);
```